components/provisioning/org.wso2.carbon.identity.provisioning/src/main/java/org/wso2/carbon/identity/provisioning/ProvisioningEntity.java

```diff
@@ -65,6 +65,18 @@ public ProvisioningEntity(ProvisioningEntityType entityTpe, ProvisioningOperatio
     }
 
     /**
+     *
+     * @param entityTpe
+     * @param operation
+     */
+    public ProvisioningEntity(ProvisioningEntityType entityTpe, ProvisioningOperation operation,
+                              Map<ClaimMapping, List<String>> attributes) {
+        this.entityType = entityTpe;
```

Comment:
Looks like that there is already a constructor that takes two arguments, which you can change it to take 3 arguments, instead of adding a new one.